astar.py

```diff
@@ -1,6 +1,10 @@
 import math
+from matplotlib import pyplot as plt
+from netCDF4 import Dataset
+import numpy as np
 
 def findPath(start, end):

```

Comment:
Another nitpick, but something to be aware of: with Python, using lowercase words separated by underscores is preferred in most contexts (`lower_case_with_underscores` instead of `camelCase`). Class names use `CapWords`. See https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions